src/state/_redux/user/actions.ts

```diff
@@ -0,0 +1,19 @@
+import { createAsyncAction } from 'typesafe-actions';
+
+export const createUser = createAsyncAction(
+    'CREATE_USER_REQUEST',
+    'CREATE_USER_SUCCESS',
+    'CREATE_USER_FAILURE',
+)<any, any, any>();
```

Comment:
Any any any :D We can create type PayloadDefault (string), and in this case it would be:
<PayloadDefault, User, PayloadDefault>